tools/iceoryx_build_test.sh

```diff
@@ -81,33 +82,55 @@ while (( "$#" )); do
         STRICT_FLAG="ON"
         shift 1
         ;;
-    "qacpp")
-        BUILD_TYPE="Release"
-        QACPP_JSON="ON"
-        shift 1
-        ;;
     "test")
         RUN_TEST=true
         TEST_FLAG="ON"
         shift 1
         ;;
-    "gcov-sonarqube")
-        COV_OUTPUT="$WORKSPACE/tools/gcov/gcovr_sonarqube.conf"
+    "dds-gateway")
+        echo " [i] Including DDS gateway in build"
+        DDS_GATEWAY_FLAG="ON"
         shift 1
         ;;
-    "with-dds-gateway")
+    "binding-c")
         echo " [i] Including DDS gateway in build"
-        DDS_GATEWAY_FLAG="ON"
+        BINDING_C_FLAG="ON"
         shift 1
         ;;
     "build-test")
         echo " [i] Building tests"
         TEST_FLAG="ON"
         shift 1
         ;;
-    "skip-introspection")
-        echo " [i] Not including introspection client in build."
-        INTROSPECTION_FLAG="OFF"
+    "roudi_env")
+        echo " [i] Building RouDi Environment"
+        ROUDI_ENV_FLAG="ON"
+        shift 1
+        ;;
+    "introspection")
+        echo " [i] Building Introspection"
+        INTROSPECTION_FLAG="ON"
+        shift 1
+        ;;
+    "buildall")
```

Comment:
What do you think of naming the arguments like the cmake flags which are activated here. Then there is one less readme the users have to read.
So if in Cmake is an option like -DBUILD_ALL=ON we have here the option `BUILD_ALL` which activates this.

Otherwise we have again inconsistencies and the user is wondering how to activate CMAKE flag XY with the shell script.

Reply:
I agree that the naming should be consistent, but we should use the naming conventions for scripts then like this: BUILD_ALL (CMake) -> build-all (script). Hope that is ok